cvat/apps/iam/filters.py

```diff
@@ -25,7 +25,18 @@ def filter_queryset(self, request, queryset, view):
         # filter isn't necessary but it is an extra check that we show only
         # objects inside an organization if the request in context of the
         # organization.
-        visibility = request.iam_context['visibility']
+        visibility = None
```

Comment:
Could you please check if we need OrganizationFilterBackend at all?

Reply:
We still need this filter, since this functionality cannot be covered by SimpleFilter. The reason of it the following:
to show valid lists in UI we want to return all non-org objects for example by requesting GET /tasks and only org-related objects with GET /tasks?org=org. And OrganizationFilterBackend filter does this.

Reply:
Why cannot SimpleFilter cover the use case? Could you please explain?

Reply:
It requires additional refactoring of SimpleFilter in order to support not only query parameters but only headers ("X-Organization")